app/services/github_service.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+require "erb"
+
+class GithubService
+  REPO = "department-of-veterans-affairs/appeals-support".freeze
+
+  def create_issue(record)
+    body = GithubIssueRenderer.new(username: record.username,
```

Comment:
I am testing the template in github_issue_renderer_spec.rb. However, I think I understand what you mean :). @shanear